src/HttpServer.gren

```diff
@@ -0,0 +1,453 @@
+effect module HttpServer where { subscription = HttpSub } exposing
+    -- Init
+    ( Permission
+    , initialize
+
+    -- Server
+    , Server
+    , ServerError(..)
+    , createServer
+
+    -- Requests
+    , Request
+    , Method(..)
+    , bodyAsString
+    , bodyFromJson
+    , toRequest
+    , requestInfo
+    , onRequest
+    )
+
+{-| Create a server that can respond to HTTP requests.
+
+You write your server using The Elm Architecture by subscribing to request
+events and responding with commands in update.
+
+Minimal example:
```

Comment:
This minimal example is a bit long. While it's welcome to see the first time one reads the documentation, I fear it can quickly become annoying when reading the docs as a reference.

I think it would be better if the actual example was removed, but that a link to the example code be left in its place.

Reply:
Agree. Removed.